src/components/VSelect/VSelect.js

```diff
@@ -175,6 +175,14 @@ export default {
     computedItems () {
       return this.filterDuplicates(this.cachedItems.concat(this.items))
     },
+    /**
+     * The range of the current input text
+     *
+     * @return {Number}
+     */
+    currentRange () {
+      return (this.selectedItem || '').length
```

Comment:
This is pretty much always `undefined` if `items` are objects. It should use `getText()` to account for this.